src/scripts/browserDeviceProfile.js

```diff
@@ -176,6 +176,13 @@ define(['browser'], function (browser) {
         return false;
     }
 
+    //temporary, until canPlayType(av1) is fixed upstream for chrome
+    function testCanPlayAv1() {
+        if (browser.chrome) {
+            return true;
+        }
```

Comment:
@dmitrylyzo This will break WebOS and Tizen if I'm not mistaken, right?

Reply:
Yeah, you're right. I'll change it to have exceptions.

Reply:
If user agent has `chrome` (Tizen 3+ and WebOS 3+ have it), `browser.chrome` is `true`. So probably yes.
How about check for `browser.versionMajor`?
I have no idea if they support AV1, so maybe we should completely exclude them.


In addition, this function has no return at the end and can be replaced with just `return ...`.

Reply:
> I have no idea if they support AV1, so maybe we should completely exclude them.

They don't, it's a fairly new format. WebOS 5 or the latest Tizen might but we'd need to check official docs.

Reply:
https://developer.samsung.com/smarttv/develop/specifications/media-specifications/2020-tv-video-specifications.html
https://ottball.com/tizen-tv-5-5-whats-new/
It looks like Tizen 5.5 (Chrome 69) supports AV1.

Maybe we should use Chrome version (`browser.chrome && browser.versionMajor >= 69`).
But according to https://caniuse.com/#feat=av1, Chrome 69 supports AV1 if it is enabled in config.

As for WebOS, this is not confirmed.
http://webostv.developer.lge.com/discover/specifications/supported-media-formats/
https://ottball.com/webos-tv-5-0/

Reply:
I've added the exceptions for Orsay, Tizen and WebOS:
I can't find any information on Orsay, so I'll assume AV1 isn't supported. 
WebOS [support for AV1 was added in version 5](http://webostv.developer.lge.com/discover/specifications/supported-media-formats/webos-50/), but only for 4k and 8k models, so to avoid breaking 1080p TVs window detection has been added.
Tizen supports AV1 as of 5.5